packages/kotti-ui/source/kotti-filters/KtFilters.vue

```diff
@@ -161,19 +161,18 @@ export default defineComponent<KottiFilters.InternalProps>({
 		}
 		const toggleListVisibility = () => {
 			isListVisible.value = !isListVisible.value
-			if (isListVisible.value) tippyInstanceRef.value.show()
+			if (isListVisible.value) tippyInstanceRef.value?.show()
```

Comment:
Should probably throw an error instead

Reply:
I think it's a must

we have it within an `onMounted > watchEffect` for the selects 

but for this, it can just suffice, to have a function that gets the `tippyRef` as an argument and throws if it doesn't exist or return it if it does